src/Commands/ArtCommand.php

```diff
@@ -19,6 +25,24 @@ public function art($name) {
     }
 
     /**
+     * If the user does not specify the $name parameter, then we will
+     * prompt for it here.
+     *
+     * @hook interact
+     */
+    public function interact(InputInterface $input, OutputInterface $output)
+    {
+        $available_art = $this->availableArt();
+
+        $io = new SymfonyStyle($input, $output);
```

Comment:
Symfony Console passes $input and $output to the interact function, and they are passed through to here. Probably best to use the parameters rather than the injected $this->input(), $this->output() and $this->io(), although in theory these should be the same objects.